scripts/kfctl.sh

```diff
@@ -235,13 +235,14 @@ ksApply () {
   fi
 
   # Create all the core components
-  ks apply default -c ambassador
-  ks apply default -c jupyterhub
-  ks apply default -c centraldashboard
-  ks apply default -c tf-job-operator
-  ks apply default -c argo
-  ks apply default -c katib
-  ks apply default -c spartakus
+  # ks apply default -c ambassador
```

Comment:
So you're changing the default deployment mechanism to deploy via the metacontroller from in cluster?
Why not leave the default as deploying client side?